c7n/resources/elb.py

```diff
@@ -106,6 +108,34 @@ def get_resources(self, resource_ids):
                 return []
             raise
 
+    def add_tags(self, client, elbs):
```

Comment:
add_tags is a bit of misnomer, ie it not adding tags to the underlying resource, its querying tags for the in memory state.